package.json

```diff
@@ -32,14 +32,14 @@
   "devDependencies": {
     "eslint": "7.1.0",
     "eslint-config-paazmaya": "7.0.0",
-    "grunt": "1.1.0",
+    "grunt": "^1.2.1",
```

Comment:
Please lock this to the given version, meaning without the prefix

Reply:
I suppose it should be `^1.1` because 1.1 is the minimal version that your package require?

Reply:
Done